main.js

```diff
@@ -0,0 +1,65 @@
+/* VARIABLES   */
+let books = [];
+const renderBooks = document.getElementById('render-books');
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const addBook = document.querySelector('#addBook');
+const localBooks = localStorage.getItem('booksStore');
+
+/* Functions */
+function render() {
+  renderBooks.innerHTML = '';
+  for (let i = 0; i < books.length; i += 1) {
+    let renderItem = '';
+    renderItem = `
+    <ul>
+      <li>${books[i].title}</li>
+      <li>${books[i].author}</li>
+      <li><a id="delete" name="${books[i].title}" href="#">Remove</a></li>
+    </ul>
+    <hr>
+    `;
+    renderBooks.innerHTML += renderItem;
+  }
+}
+
+function getStore() {
+  if (localBooks) {
+    books = JSON.parse(localBooks);
+  }
+}
+
+function setStore() {
+  localStorage.setItem('booksStore', JSON.stringify(books));
+}
+
+function addNewBook() {
+  const newBook = {
+    title: titleInput.value,
+    author: authorInput.value,
+  };
+  books.push(newBook);
+  setStore();
+  titleInput.value = '';
+  authorInput.value = '';
+  render();
+}
+
+function deleteBook(e) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.